src/dispatch_common.c

```diff
@@ -834,3 +834,13 @@ WRAPPER(epoxy_glEnd)(void)
 
 PFNGLBEGINPROC epoxy_glBegin = epoxy_glBegin_wrapped;
 PFNGLENDPROC epoxy_glEnd = epoxy_glEnd_wrapped;
+
+epoxy_resolver_failure_handler_t epoxy_resolver_failure_handler;
+
+epoxy_resolver_failure_handler_t
+epoxy_set_resolver_failure_handler(epoxy_resolver_failure_handler_t handler)
+{
+    epoxy_resolver_failure_handler_t old = epoxy_resolver_failure_handler;
```

Comment:
Updated the branch to fix this (also made the typedefs a bit more self-documenting)